components/portal/misc/partials/frame-page.html

```diff
@@ -29,9 +29,13 @@ <h1 class="md-display-1" ng-if="pageTitle">
       </h1>
       <add-to-home ng-if="appShowAddToHome"></add-to-home>
     </div>
-    <div class="content-wrapper__push-content" ng-if="menuPushContent">
+
+    <span id="skip-to-here"></span>
```

Comment:
An interesting approach :thinking: 
I've usually seen `skip-to-here` applied directly to a `<main>` tag.
Progress is progress :+1:

Reply:
Yup. And *not* applying it directly to `<main>` probably means we're not getting `<main>` quite right. Which might be a bigger lift to fix.

At this point the "Beta Settings"-ness of the Beta Settings page is reflected in the page title, so arguably it's redundant to also say that in the main content. Figure anyone following the skip-to-main content link already knows they're on the Beta Settings page or at least could have known that if they wanted to and is ready to skip to the actual page content.

At least that's the hypothesis. I'm not sure this change is really going to help, but the inadequacy of our skip-to-main-content link was flagged in a (manual) a11y testing, so trying to do something about it. Will change and re-test.